modules/rosa-cluster-autoscaler-ui-settings.adoc

```diff
@@ -16,25 +18,45 @@ The tables explain all the configurable UI settings when using cluster autoscali
 |Type or Range
 |Default
 
-|`balanceSimilarNodeGroups`
+|`balance-similar-node-groups`
 |If `true`, this setting automatically identifies node groups with the same instance type and the same set of labels and tries to keep the respective sizes of those node groups balanced.
 |`boolean`
 |false
 
-|`logVerbosity`
-|Sets the autoscaler log level. Default value is 1, level 4 is recommended for debugging and level 6 enables almost everything.
-|`integer`
-|1
+|`skip-nodes-with-local-storage`
+|If `true`, the cluster autoscaler never deletes nodes with pods with local storage, e.g. EmptyDir or HostPath.
+|`boolean`
+|true
 
-|`maxPodGracePeriod`
+|`max-pod-grace-period (max-graceful-termination-sec)`
 |Gives pods graceful termination time before scaling down.
 |`integer`
 |600
 
-|`skipNodesWithLocalStorage`
-|If `true`, the cluster autoscaler never deletes nodes with pods with local storage, e.g. EmptyDir or HostPath.
+|`max-node-provision-time`
+|Maximum time the cluster autoscaler waits for nodes to be provisioned.
+|`string`
+|15m
+
+|`pod-priority-threshold`
```

Comment:
`pod-priority-threshol` is the only other parameter that is renamed between API and k8s. In the UI, we show `pod-priority-threshold (expendable-pods-priority-cutoff)`

Reply:
We will only show `pod-priority-threshold`